src/moderation/customFilter.ts

```diff
@@ -15,7 +15,19 @@ function customWordFilter(message: Message, guildConfig: Guild) {
 }
 
 function customRegexFilter(message: Message, guildConfig: Guild) {
-
+    for (const words of message.content.split(" ")) {
```

Comment:
I do not think you need to split the message when testing a regex against a sentence, you should be able to remove it for higher performance and code efficiency

Reply:
oh right